sdks/client-csharp/ConsoleApp1/README.md

```diff
@@ -0,0 +1,5 @@
+# Console Example
+
+This is a simple console example for FeatureHub. Replace the URL with one from your own server
+or from DemoHub (the Try Now button at [FeatureHub](https://featurehub.io))
```

Comment:
perhaps don't mention DemoHub (it's only us who know it.) Better to say "from the demo app that can be accessed from the featurehub.io"

Reply:
good point